### ISSUE TITLE
--include option not working

### ISSUE THREAD

**Issue by chengwei45254** [hrcorval/behavex#48]
**Describe the bug**
When invoking behavex with --include option followed by a feature file name, the following error occurred:
_json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)_

**To Reproduce**
1. Install behavex
2. Run the following command:    
behavex --include "some_feature_file.feature"
3. The above said error with stack trace is displayed (see screenshot below)

**Expected behavior**
Like invoking original behave command, when --include option followed by a feature file name are given, behavex should run the given feature file successfully.

**Screenshots**
![image](https://user-images.githubusercontent.com/47975135/195261248-24bd24da-9346-495f-937b-9de3cd8ab9ef.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome was used for the feature test, but the error occurred before the test could be run
 - Version: Windows 10 21H2

**Smartphone (please complete the following information):**
N/A

**Additional context**
N/A

**Comment by chengwei45254** [comment]
Hi @hrcorval, thanks for your work on behavex. It's a useful behave wrapper. 
  
For this --include issue I reported, I've spent some time tracing the code, and found that it seems to relate to the following piece of code in _set_env_variables() of runner.py:

![image](https://user-images.githubusercontent.com/47975135/195265359-381c257b-860d-44b1-affd-17e22da53e07.png)

I'm not sure the purpose of the json calls, but if I removed those json calls, it will seem to work fine:

![image](https://user-images.githubusercontent.com/47975135/195265863-ccd42dea-eca3-4901-b886-7632f71f99f2.png)

Not sure if these findings are helpful.

Thanks.
